contracts/strategies/StrategyEthHegicLP.sol

```diff
@@ -101,13 +107,16 @@ contract StrategyEthHegicLP is BaseStrategy {
     }
 
 
-    // adjusts position. Will not deposit if timelock check fails.
+    // adjusts position.
     function adjustPosition(uint256 _debtOutstanding) internal override {
        //emergency exit is dealt with in prepareReturn
         if (emergencyExit) {
           return;
        }
 
+        // check the withdraw flag first before depositing anything
+        require (withdrawFlag == false, "!vault withdrawing");
```

Comment:
failing the tx feels like too much. just return like we do with `emergencyExit` some lines above.